frontend-nx/apps/stujo/styles/globals.css

```
@@ -0,0 +1,717 @@
+    var(--stujo-primary),
+    var(--stujo-gradient-dark)
+  );
+  --stujo-text: #333333;
+  --stujo-bg: #ffffff;
+  --stujo-muted: #646363;
+  --stujo-border: #dedad5;
+  --stujo-divider: #737373;
+  --stujo-error: #ed2507;
+  /* Lato is declared like on the live site; its webfont 404s there too, so
+     the effective rendering is the same fallback stack. */
+  --stujo-font: Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Lato" to be "lato" (value-keyword-case)

(value-keyword-case)